db/pgstorage/migrations/0004.sql

```diff
@@ -0,0 +1,95 @@
+-- +migrate Down
+
+DROP TABLE IF EXISTS sync.monitored_txs;
+
+ALTER TABLE sync.deposit DROP COLUMN ready_for_claim;
+
+ALTER TABLE sync.block DROP CONSTRAINT block_hash_unique;
+
+DROP INDEX IF EXISTS mt.rht_key_idx;
+
+-- +migrate Up
+
+ALTER TABLE
+    sync.deposit
+ADD
+    COLUMN ready_for_claim BOOLEAN NOT NULL DEFAULT FALSE;
+
+ALTER TABLE
+    sync.block
+ADD
+    CONSTRAINT block_hash_unique UNIQUE (block_hash);
+
+CREATE TABLE sync.monitored_txs (
+    id BIGINT PRIMARY KEY,
+    block_id BIGINT REFERENCES sync.block (id) ON DELETE CASCADE,
+    from_addr BYTEA NOT NULL,
+    to_addr BYTEA,
+    nonce BIGINT NOT NULL,
+    value VARCHAR,
+    data BYTEA,
+    gas BIGINT NOT NULL,
+    status VARCHAR NOT NULL,
+    history BYTEA [],
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+UPDATE mt.root SET deposit_cnt = deposit_cnt - 1;
```

Comment:
this was already included in v0.0.3 and you don't rollback it in the migration down